src/components/home.js

```diff
@@ -1,39 +1,53 @@
+/* eslint-disable no-unused-expressions */
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { NavLink } from 'react-router-dom';
+import { ArrowRightCircle } from 'react-bootstrap-icons';
 import { loadCountryDetail, getCountry } from '../redux/home';
-// import Countries from '../redux/countries';
-import Details from './details';
+import style from './style.module.css';
+import coronavirus from '../assets/coronavirus.jpg';
 
 export default function Home() {
   const state = useSelector((state) => state.homeReducer.covid_data);
   const dispatch = useDispatch();
-  console.log(state);
 
+  const dark = () => {
+    const con = document.querySelectorAll('.country');
+    for (let j = 0; j < con.length; j += 4) {
+      if (j % 2 === 0) {
+        j + 1 < con.length && (con[j + 1].classList.add('dark'));
+        j + 2 < con.length && (con[j + 2].classList.add('dark'));
+      }
+    }
+  };
   useEffect(() => {
     if (state.length === 0) {
       loadCountryDetail().then((data) => {
-        console.log(data);
         dispatch(getCountry(data));
-      });
+      }).then(
+        dark,
+      );
     }
+    dark();
   }, []);
 
   return (
     <div>
-      <div>
-        <div>
+      <div className={style.intro}>
```

Comment:
- In the home component it is expected of us to add a stateless filter component that can be used to filter the data on the homepage.

This was hinted at in the **_`Description`_** of the project. See the quotation below:

---
> one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
---

Then, under **_`Interactions`_** section, under the sub-title **_`Home page`_**, this requirement is mentioned. See quote below:

---
### Home page
- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
---


### Explanation
I see you tried to implement filtering in the details component. That was a good attempt. But, all the actions for this requirement happen on the homepage.

Here is what is meant by filtering:

Think of a scenario where your app loads thousands of items on the homepage for the user to select, it would be tedious to scroll through all that to get to the particular one the user is interested in. Therefore, it would be nice to have something like a search filed, in which the user can type the name of the item he/she is interested in. The app will then filter the items based on what the user types and then only display that item on the homepage for the user to select before navigating to the details page.